### PR TITLE
Typo fixes

### PR DESCRIPTION
**Pull request by nexxai**
Was bored on my lunch break so spent some time typo-hunting

### CODE REVIEW

**Comment by PathToLife** [comment]
Not sure about merging 74 odd different files among 100+ pull requests?
I agree spelling errors should be fixed, though the docs say otherwise...

https://github.com/microsoft/vscode/wiki/How-to-Contribute#spell-check-errors

>Pull requests only fixing spell check errors in source code are not recommended

>make sure it doesn't touch multiple feature areas
Perhaps try a smaller pull request first on strings / comments only.

File: extensions/fsharp/syntaxes/fsharp.tmLanguage.json [hunk 1]
```diff
@@ -1265,7 +1265,7 @@
 							"match": "\\\\([\\\\''ntbr]|x[a-fA-F0-9]{2}|u[a-fA-F0-9]{4}|u[a-fA-F0-9]{8})"
 						},
 						{
-							"name": "invalid.illeagal.character.string.fsharp",
+							"name": "invalid.illegal.character.string.fsharp",
```
Comment (JacksonKearl) [inline extensions/fsharp/syntaxes/fsharp.tmLanguage.json:1268]: no *.tmLanguage.json files should change, they're contributed from upstream repositories. See start of file.

File: extensions/git/src/timelineProvider.ts [hunk 2]
```diff
@@ -200,7 +200,7 @@ export class GitTimelineProvider implements TimelineProvider {
 			if (working) {
 				const date = new Date();
 
-				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommitedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
+				const item = new GitTimelineItem('', index ? '~' : 'HEAD', localize('git.timeline.uncommittedChanges', 'Uncommitted Changes'), date.getTime(), 'working', 'git:file:working');
```
Comment (JacksonKearl) [inline extensions/git/src/timelineProvider.ts:203]: Changing the keys in localize calls will force the entries to be relocalized and users will loose thir translations until the new localizations come in. So please don't change these.

File: extensions/python/syntaxes/MagicRegExp.tmLanguage.json [hunk 3]
```diff
@@ -160,7 +160,7 @@
 				}
 			]
 		},
-		"regexp-charecter-set-escapes": {
+		"regexp-character-set-escapes": {
```
Comment (aeschli) [inline extensions/python/syntaxes/MagicRegExp.tmLanguage.json:163]: Part of an imported grammar, needs to be fixed in the grammar repo

File: extensions/python/syntaxes/MagicPython.tmLanguage.json [hunk 4]
```diff
@@ -1951,7 +1951,7 @@
 				}
 			]
 		},
-		"regexp-charecter-set-escapes": {
+		"regexp-character-set-escapes": {
```
Comment (aeschli) [inline extensions/python/syntaxes/MagicPython.tmLanguage.json:1954]: Part of an imported grammar, needs to be fixed in the grammar repo

**Review by shivaylamba — APPROVED** [review]

**Review by unrealapex — APPROVED** [review]

**Pull request closed without merging**